x-pack/plugins/observability/public/pages/rules/rules_tab.tsx

```diff
@@ -97,6 +98,7 @@ export function RulesTab({ setRefresh, stateRefresh }: RulesTabProps) {
       onSearchFilterChange={handleSearchFilterChange}
       onStatusFilterChange={handleStatusFilterChange}
       onTypeFilterChange={handleTypeFilterChange}
+      initialSelectedConsumer={AlertConsumers.LOGS}
```

Comment:
@XavierM When I change this to `initialSelectedConsumer={AlertConsumers.INFRASTRUCTURE}`, should I see the `Metrics` as the default option? For me, it still shows `Logs` as the default option.

Reply:
You also need to change it there -> https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability/public/pages/rules/rules.tsx#L150 because you folks are by passing our rule creation button.